docs/graphqlserve/graphqlserve.md

```diff
@@ -0,0 +1,171 @@
+---
+id: graphqlserve
+title: Running an API without code
+sidebar_label: API without code
+---
+
+With GraphQL Serve you can have a full featured [GraphQL CRUD](https://graphqlcrud.org/) API with subscriptions and data synchronization running in just a few seconds without writing a single line of code - all you need is a data model `.graphql` file.
```

Comment:
Name with -. No strong opinion

Reply:
You mean `GraphQL-Serve`?

Reply:
I prefer `GraphQL Serve` not `GraphQL-Serve` when using the full name like this.

Reply:
Adding the hyphen looks strange.

Reply:
Yeap. Me too, I asked to see which name @wtrocki was referring too.